src/react-apps/applications/service-development/__tests__/Administration.test.tsx

```diff
@@ -95,17 +98,19 @@ describe('HandleMergeConflictAbort', () => {
         serviceDescription={mockServiceDescription}
         serviceDescriptionIsSaving={mockServiceDescriptionIsSaving}
         initialCommit={mockInitialCommit}
+        serviceId={mockServiceId}
+        serviceIdIsSaving={mockServiceIdIsSaving}
       />,
     );
 
     const instance = wrapper.instance() as AdministrationComponent;
 
     expect(mockFetchService).toHaveBeenCalled();
     expect(mockFetchInitialCommit).toHaveBeenCalled();
-    expect(mockFetchServiceName).toHaveBeenCalled();
     expect(mockFetchServiceDescription).toHaveBeenCalled();
     expect(instance.state.serviceName).toEqual(mockServiceName);
     expect(instance.state.serviceDescription).toEqual(mockServiceDescription);
+    expect(instance.state.serviceId).toEqual(mockServiceId);
```

Comment:
Is it possible to have assertions before the dispatchers to see if the state do change?

Reply:
Not that I can see, instance is based on the wrapper that does the mount. So the assertions has to be done before the mount. Can't see how that can be possible